app/api/src/crud/crud_static_layer.py

```diff
@@ -12,7 +12,27 @@ async def drop_postgis_table(self, db: AsyncSession, table_name: str):
         table = Table(table_name, metadata_obj)
         await db.execute(DropTable(table, if_exists=True))
 
-    pass
+    async def list_static_layer_table_names(self, db: AsyncSession, name_like=""):
+        metadata_obj = MetaData(schema="information_schema")
+        tables = Table(
```

Comment:
A very elegant solution to avoid SQL-string. I did not know this works.

Reply:
Thanks a lot @EPajares . Also while searching about functions I found that we can use SqlAlechemy `func` method to call the functions ([ref](https://docs.sqlalchemy.org/en/14/core/functions.html)). So we can use it to have user defined functions also as SqlAlchemy Query object and use it's other methods cleaner.

> Note that any name not known to func generates the function name as is - **there is no restriction on what SQL functions can be called, known or unknown to SQLAlchemy, built-in or user defined**. The section here only describes those functions where SQLAlchemy already knows what argument and return types are in use.